packages/reakit/src/_utils/createElementRef.ts

```diff
@@ -1,5 +1,9 @@
 /* eslint-disable no-param-reassign */
-const createElementRef = (scope, property) => element => {
+import * as React from "react";
+
+const createElementRef = (scope: any, property: string) => (
```

Comment:
`scope` is a `React.ComponentClass` instance (or something similar). `property` is a key of that instance (i.e. `this[property]`).

I'm not sure how to type it. Maybe something like this:
```ts
function createElementRef<T extends React.ComponentClass<any>>(scope: T, property: keyof T) => 
```

If it gets too complicated, it's Ok to keep it as `any` and `string`.

Reply:
Also, I don't know if `React.createRef` will work here without any changes. Have you tested it?